http/headers/Origin.json

```diff
@@ -16,10 +16,16 @@
               "version_added": "1"
             },
             "chrome_android": "mirror",
-            "edge": {
-              "version_added": "12",
-              "notes": "Before Edge 79, this header was not sent with `POST` requests."
-            },
+            "edge": [
+              {
+                "version_added": "79"
+              },
+              {
+                "version_added": "12",
```

Comment:
```suggestion
                "version_added": "12",
                "version_removed": "79",
```

Reply:
Should I make the same change to the equivalent Firefox entry?

Reply:
Done for both